src/components/Services.jsx

```diff
@@ -2,43 +2,66 @@ import React from 'react'
 import sortBy from 'lodash/sortBy'
 import { connect } from 'react-redux'
 import { withClient } from 'cozy-client'
+import { queryConnect } from 'cozy-client'
 import { translate } from 'cozy-ui/react/I18n'
 import keyBy from 'lodash/keyBy'
 import has from 'lodash/has'
+import flow from 'lodash/flow'
 
 import AddServiceTile from 'components/AddServiceTile'
 import KonnectorTile from 'components/KonnectorTile'
-import NoServicesList from 'components/NoServicesList'
-import classNames from 'classnames'
+import CandidateCategoryTile from 'components/CandidateCategoryTile'
+import CandidateServiceTile from 'components/CandidateServiceTile'
+import EmptyServicesListTip from 'components/EmptyServicesListTip'
 import { getInstalledKonnectors } from 'reducers/index'
 import useAppsInMaintenance from 'hooks/withAppsInMaintenance'
+import candidatesConfig from 'config/candidates'
 
-export const Services = ({ t, installedKonnectors, client }) => {
+export const Services = ({
+  t,
+  installedKonnectors,
+  konnectorSuggestions,
```

Comment:
it seems weird to have one side : `installedKonnectors` and the other side `konnectorSuggestions`. Does renaming `konnectorSuggestions` to `suggestedKonnectors` make sense?

Reply:
Oh in fact `konnectorSuggestions` is the result of a queryConnect, I'm wondering if should not tell it in its name. When reading the component first time, I thought that both props are the same shape, but nop :(